Makefile

```diff
@@ -2,8 +2,7 @@ MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 .DEFAULT_GOAL := help
 DOCKER_COMPOSE = docker-compose
-DOCKER ?= $(shell which podman 2> /dev/null || which docker 2> /dev/null || echo "docker")
-S2I = script/s2i
+DOCKER ?= $(shell which docker 2> /dev/null || echo "docker")
```

Comment:
This was done because Podman requires to have a running container in order to mount the volume from it (see https://github.com/3scale/APIcast/pull/1344/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R56). Should we consider leaving the `-source` container running (instead of [stopping it](https://github.com/3scale/APIcast/pull/1344/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R89)) to keep Podman support or would that potentially cause other side-effects?

Reply:
not sure that podman works, even leaving the container running.  

That said, the root issue comes from the fact that in the `docker-compose.yaml` service `prove` has `volumes_from` which requires a running(podman) or stopped (docker) container. With just a volume should be enough. But a container is required.

The service `prove`  in the   `docker-compose.yaml` is only used in makefile's target `prove-docker`. Which is an alternative way, not the main one in the development workflow, to run integration tests.  I plan to fix this and have `prove-docker` use a volume, instead of a volume in a (running or stopped) container. But this PR is already big enough and that deserves a new PR.

Reply:
understood, thanks for clarifying